x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/hooks/index.tsx

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
similar to my comment on core context, we probably don't need this EPM-specific hooks any more either